tauri-app/src-tauri/src/commands/fork.rs

```diff
@@ -0,0 +1,13 @@
+use rain_orderbook_common::fork::fork_parse_dotrain;
+
+#[tauri::command]
+pub async fn fork_parse(
+    rainlang: &str,
+    front_matter: &str,
+    fork_url: &str,
+    fork_block_number: u64,
+) -> Result<(), String> {
+    fork_parse_dotrain(rainlang, front_matter, fork_url, fork_block_number)
+        .await
+        .map(|_| ())
```

Comment:
do we want to return the error?

Reply:
this command is specifically designed for text editor, which needs string as the error to show it on text editor

Reply:
if you'd like you need to make a new wrapper command that returns the Error itself, but this one as is used in text editor should return the error as string so text editor can consume it

Reply:
Yes that's the idea -- use the CommandError type to make a wrapper error. That error type can be serialized and displayed in the editor.